test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -72,9 +72,60 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     assert_not page.has_content?("#{benchmark_run_category_humanize} memory Graph")
   end
 
+  test "User should be able to view long running benchmark graphs when they go back and forward 
+    in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
+
+    benchmark_run = benchmark_runs(:array_iterations_run2)
+    benchmark_run2 = benchmark_runs(:array_count_run)
+
+    visit '/ruby/ruby/commits'
+
+    assert page.has_content?(
+      I18n.t('repos.show.title', repo_name: benchmark_run.initiator.repo.name.capitalize)
+    )
+    assert page.has_content?(I18n.t('repos.show.select_benchmark'))
+
+    within "form" do
+      choose(benchmark_run.benchmark_type.category)
+      choose(benchmark_run2.benchmark_type.category)
+    end
+
+    benchmark_run_category_humanize = benchmark_run.benchmark_type.category.humanize
+    benchmark_run2_category_humanize = benchmark_run2.benchmark_type.category.humanize
+    
+    assert page.has_css?(".chart .highcharts-container")
+    assert page.has_content?("#{benchmark_run2_category_humanize} Graph")
+    assert_not page.has_content?("#{benchmark_run2_category_humanize} memory Graph")
+
+    page.evaluate_script('window.history.back()')
+    
+    assert page.has_content?("#{benchmark_run_category_humanize} Graph")
+    assert page.has_content?("#{benchmark_run_category_humanize} memory Graph")
+    assert page.has_content?("#{benchmark_run_category_humanize} Script")
+    
+
```

Comment:
yea just one blank line will do :) I generally separate `assertions` from `actions` with one blank line. Helps with transitioning when trying to understand what the code is doing with each block. Haha I really should do up a style guide xD